log/scope.go

```diff
@@ -302,6 +310,9 @@ func (s *Scope) Debugf(args ...interface{}) {
 		if len(args) > 1 {
 			msg = fmt.Sprintf(msg, args[firstIdx+1:]...)
 		}
+		if !s.isSampled(msg) {
```

Comment:
is it intentionally only at debug level?

Reply:
good catch - i forgot to remove that when I moved the callsite.